Fenced_Frames_Ads_Reporting.md

```diff
@@ -50,7 +50,7 @@ The following new APIs will be added for achieving this.
 
 There are two variants of the `reportEvent` API for event-level reporting that are invoked with different sets of parameters. In the first variant, fenced frames can invoke the `reportEvent` API to tell the browser to send a beacon with event data to a URL registered by the worklet in `registerAdBeacon` (see below). Depending on the declared `destination`, the beacon is sent to either the buyer's or the seller's registered URL. Examples of such events are mouse hovers, clicks (which may or may not lead to navigation e.g. video player control element clicks), etc.
 
-This API is available from all documents in a fenced frame that are same-origin to the mapped URL of the fenced frame config (i.e., the ad creative URL that won the Protected Audience auction).
+This API is available from all documents in a fenced frame tree (i.e., the ad creative URL that won the Protected Audience auction). Documents that are same-origin to the mapped URL of the fenced frame config can call this API without any restrictions. Cross-origin documents can only call this API if there is opt-in from both the fenced frame root and the cross-origin document. The fenced frame root opts in by being served with a new `Allow-Cross-Origin-Event-Reporting` response header set to `true`. The cross-origin document opts in by calling `reportEvent` with `crossOriginExposed=true`.
```

Comment:
Let's also link the related GH issue so the motivation for this change is clear.